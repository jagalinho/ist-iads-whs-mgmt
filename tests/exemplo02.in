a 60fdba63 4
a c614e44d 149
a ff4095a1 38
a 60fdba63 -5
l
r c614e44d
a 90adba63 4
l
x
