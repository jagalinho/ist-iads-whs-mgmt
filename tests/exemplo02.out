60fdba63 0
c614e44d 149
ff4095a1 38
60fdba63 0
90adba63 4
ff4095a1 38
3
